acl: fix the order of expected output of getfacl

The result of getfacl is sorted by user id.
In Centos or RHEL, bin user id is 1 and daemon user id is 2.
But in our image, bin user id is 2 and daemon user id is 1.
The patch fixes this issue to make ptest pass.

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Chong Lu <Chong.Lu@windriver.com>
---
 test/misc.test |   38 +++++++++++++++++++-------------------
 1 file changed, 19 insertions(+), 19 deletions(-)

diff --git a/test/misc.test b/test/misc.test
index 6e98053..53ae5b0 100644
--- a/test/misc.test
+++ b/test/misc.test
@@ -79,8 +79,8 @@ Multiple users
 
 	$ getfacl --omit-header f
 	> user::rw-
-	> user:bin:rw-
 	> user:daemon:r--
+	> user:bin:rw-
 	> group::r--
 	> mask::rw-
 	> other::r--
@@ -94,8 +94,8 @@ Multiple groups
 
 	$ getfacl --omit-header f
 	> user::rw-
-	> user:bin:rw-
 	> user:daemon:r--
+	> user:bin:rw-
 	> group::r--
 	> group:daemon:r--
 	> group:users:rw-
@@ -111,8 +111,8 @@ Remove one group
 
 	$ getfacl --omit-header f
 	> user::rw-
-	> user:bin:rw-
 	> user:daemon:r--
+	> user:bin:rw-
 	> group::r--
 	> group:daemon:r--
 	> mask::rw-
@@ -146,8 +146,8 @@ Default ACL
 
 	$ getfacl --omit-header d
 	> user::rwx
-	> user:bin:rwx
 	> user:daemon:rw-
+	> user:bin:rwx
 	> group::r-x
 	> mask::rwx
 	> other::---
@@ -236,16 +236,16 @@ Add some users and groups
 
 	$ getfacl --omit-header d/d
 	> user::rwx
-	> user:bin:rwx	#effective:r-x
 	> user:daemon:r-x
+	> user:bin:rwx	#effective:r-x
 	> group::r-x
 	> group:daemon:rwx	#effective:r-x
 	> group:users:r-x
 	> mask::r-x
 	> other::---
 	> default:user::rwx
-	> default:user:bin:rwx	#effective:r-x
 	> default:user:daemon:r-x
+	> default:user:bin:rwx	#effective:r-x
 	> default:group::r-x
 	> default:mask::r-x
 	> default:other::---
@@ -262,16 +262,16 @@ Symlink in directory with default ACL?
 
 	$ getfacl --omit-header d/l
 	> user::rwx
-	> user:bin:rwx	#effective:r-x
 	> user:daemon:r-x
+	> user:bin:rwx	#effective:r-x
 	> group::r-x
 	> group:daemon:rwx	#effective:r-x
 	> group:users:r-x
 	> mask::r-x
 	> other::---
 	> default:user::rwx
-	> default:user:bin:rwx	#effective:r-x
 	> default:user:daemon:r-x
+	> default:user:bin:rwx	#effective:r-x
 	> default:group::r-x
 	> default:mask::r-x
 	> default:other::---
@@ -287,16 +287,16 @@ Does mask manipulation work?
 
 	$ getfacl --omit-header d/d
 	> user::rwx
-	> user:bin:r-x
 	> user:daemon:r-x
+	> user:bin:r-x
 	> group::r-x
 	> group:daemon:r-x
 	> group:users:r-x
 	> mask::r-x
 	> other::---
 	> default:user::rwx
-	> default:user:bin:rwx	#effective:r-x
 	> default:user:daemon:r-x
+	> default:user:bin:rwx	#effective:r-x
 	> default:group::r-x
 	> default:mask::r-x
 	> default:other::---
@@ -308,16 +308,16 @@ Does mask manipulation work?
 
 	$ getfacl --omit-header d/d
 	> user::rwx
-	> user:bin:r-x
 	> user:daemon:r-x
+	> user:bin:r-x
 	> group::r-x
 	> group:daemon:r-x
 	> group:users:r-x
 	> mask::r-x
 	> other::---
 	> default:user::rwx
-	> default:user:bin:rwx
 	> default:user:daemon:r-x
+	> default:user:bin:rwx
 	> default:group::r-x
 	> default:mask::rwx
 	> default:other::---
@@ -333,8 +333,8 @@ Remove the default ACL
 
 	$ getfacl --omit-header d
 	> user::rwx
-	> user:bin:rwx
 	> user:daemon:rw-
+	> user:bin:rwx
 	> group::r-x
 	> mask::rwx
 	> other::---
@@ -373,14 +373,14 @@ Now, chmod should change the group_obj entry
 
 	$ getfacl --omit-header d
 	> user::rwx
-	> user:bin:r-x
 	> user:daemon:rwx
+	> user:bin:r-x
 	> group::rwx
 	> mask::rwx
 	> other::r-x
 	> default:user::rwx
-	> default:user:bin:r-x
 	> default:user:daemon:rwx
+	> default:user:bin:r-x
 	> default:group::rwx
 	> default:mask::rwx
 	> default:other::r-x
@@ -392,14 +392,14 @@ Now, chmod should change the group_obj entry
 
 	$ getfacl --omit-header d
 	> user::rwx
-	> user:bin:r-x
 	> user:daemon:rwx	#effective:r-x
+	> user:bin:r-x
 	> group::rwx	#effective:r-x
 	> mask::r-x
 	> other::---
 	> default:user::rwx
-	> default:user:bin:r-x
 	> default:user:daemon:rwx
+	> default:user:bin:r-x
 	> default:group::rwx
 	> default:mask::rwx
 	> default:other::r-x
@@ -411,14 +411,14 @@ Now, chmod should change the group_obj entry
 
 	$ getfacl --omit-header d
 	> user::rwx
-	> user:bin:r-x
 	> user:daemon:rwx	#effective:r-x
+	> user:bin:r-x
 	> group::rwx	#effective:r-x
 	> mask::r-x
 	> other::---
 	> default:user::rwx
-	> default:user:bin:r-x
 	> default:user:daemon:rwx
+	> default:user:bin:r-x
 	> default:group::rwx
 	> default:mask::rwx
 	> default:other::r-x
-- 
1.7.9.5

